### DISCUSSION TITLE
Sigma File Manager v2.0.0-alpha.1 preview

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [aleksey-hoffman/sigma-file-manager#313] [Announcements]
![v2-announcement-wide](https://github.com/aleksey-hoffman/sigma-file-manager/assets/61761672/140df973-9706-4c96-ada6-6fa73de199d9)

> [!NOTE]
> ### Early preview release
> Most features weren't ported from v1 yet. Check out discussion for this release to leave your feedback

# Download

See the assets section below to download the installer for your platform.

# Release Notes

- Migrated the app from Electron to Tauri framework
- App installation file size reduced from `153 MB` to `4 MB` (Windows)
- App installed size reduced from `419 MB` to `12 MB` (Windows)
- 1 new major feature

## New major features
- **Navigator:** added resizable panes feature, which lets you split the navigator view in half and work with 2 directories side by side.

## Improvements
- **UX:** App installation file size reduced from `153 MB` to `4 MB` (Windows)
- **UX:** App installed size reduced from `419 MB` to `12 MB` (Windows)

## Development
- **Architecture:** Migrated the app from Electron to Tauri framework.
- **Code refactor:** code quality was improved a lot.

# Support

Consider supporting my work on Patreon (monthly, with [rewards](https://patreon.com/sigma_file_manager)):

<a target="_blank" href="https://patreon.com/sigma_file_manager">
  <img
    src="https://raw.githubusercontent.com/aleksey-hoffman/sigma-file-manager/main/.github/media/patreon_button.png"
    width="164px"
    style="box-shadow: 0px 6px 24px rgb(255, 66, 77, 0.2); margin: 16px 0"
  />
</a>

<hr /><em>This discussion was created from the release <a href='https://github.com/aleksey-hoffman/sigma-file-manager/releases/tag/v2.0.0-alpha.1'>Sigma File Manager v2.0.0-alpha.1 preview</a>.</em>

**Comment by bassepeder** [comment]
I would recommend creating a new PR label, something like "v2", in order to quickly distinguish issues directly related to this version. A lot of issues will not be about reporting a bug, but simply about features not being implemented at all.

**Reply by aleksey-hoffman** [reply]
Thanks for the suggestion mate

**Comment by wushutune** [comment]
When I checked the "v2" version, I found that it uninstalled all the software in my installation directory.

**Comment by brunocpazini** [comment]
It would be great to incorporate metadata into files and directories, as well as implement version control and establish clear naming conventions for files and directories. Imagine that, when developing a new project, you would create hierarchical diagrams of the folders and add guidelines for naming files and folders based on the incorporated metadata.
